client/components/site-mockup/index.jsx

```diff
@@ -0,0 +1,67 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+function MockupChromeMobile() {
+	return (
+		<div className="site-mockup__chrome-mobile">
+			<span className="site-mockup__chrome-label">
+				{ translate( 'Phone', {
+					comment: 'Label for a phone-sized preview of a website',
+				} ) }
+			</span>
+		</div>
+	);
+}
+
+function MockupChromeDesktop() {
+	return (
+		<div className="site-mockup__chrome-desktop">
+			<svg width="38" height="10">
+				<g>
+					<rect width="10" height="10" rx="5" />
+					<rect x="14" width="10" height="10" rx="5" />
+					<rect x="28" width="10" height="10" rx="5" />
+				</g>
+			</svg>
+			<span className="site-mockup__chrome-label">{ translate( 'Website Preview' ) }</span>
+		</div>
+	);
+}
+
+const SiteMockup = React.memo(
+	( { className, children, siteStyle, siteType, size = 'mobile', tagline, title } ) => (
```

Comment:
@spen Given the use of `SiteMockup` here is much simpler than how this component was previously used, what do you think of removing `siteStyle`, `siteType`, `tagline`, and `title` and keep this just as the "browser chrome" around an image?

Reply:
That makes sense to me! We could cut down on a lot of this without those props :)